js/appconfig.ts

```diff
@@ -82,6 +82,7 @@ export class AppConfig {
         isDesktopApp: false,
         realMoneyCurrencySymbol: "$",
         gameMoneySymbol: "",
+        expandActionBlockIfNotYourTurn: true,
```

Comment:
Rename to `allowExpandActionBlockGuestureOnlyOnMyTurn` and default value would be `false`